core/core.gradle.kts

```diff
@@ -1,6 +1,7 @@
 val libs: (String) -> String by rootProject.ext
 
 dependencies {
+    api("org.apache.httpcomponents:httpclient:4.5.12")
```

Comment:
1. It should be `testImplementation`.
2. Please move this notation to `dependencies.gradle.kts`, then use `testImplementation(libs("httpclient"))` here, just like other dependencies.

Reply:
'kay, will do.